fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -153,6 +154,61 @@ public void testCreateVersion() throws Exception {
         }
     }
 
+    @Test
+    public void testCreateVersionWithSlugHeader() throws Exception {
+        createVersionedContainer(id, subjectUri);
+
+        // Bad request with Slug header to create memento
+        final String mementoDateTime = "Tue, 3 Jun 2008 11:05:30 GMT";
+        final String body = "<" + subjectUri + "> <info:test#label> \"bar\"";
+        final HttpPost post = postObjMethod(id + "/" + FCR_VERSIONS);
+
+        post.addHeader("Slug", "version_label");
+        post.addHeader(MEMENTO_DATETIME_HEADER, mementoDateTime);
+        post.addHeader(CONTENT_TYPE, "text/n3");
+        post.setEntity(new StringEntity(body));
+
+        assertEquals("Created memento with Slug!",
+                BAD_REQUEST.getStatusCode(), getStatus(post));
+    }
+
+    @Test
+    public void testCreateVersionWithMementoDatetimeFromat() throws Exception {
+        createVersionedContainer(id, subjectUri);
+
+        // Create memento with RFC-1123 date-time format
+        final String mementoDateTime = "Tue, 3 Jun 2008 11:05:30 GMT";
+        final String body = "<" + subjectUri + "> <info:test#label> \"bar\"";
+        final HttpPost post = postObjMethod(id + "/" + FCR_VERSIONS);
+
+        post.addHeader(MEMENTO_DATETIME_HEADER, mementoDateTime);
+        post.addHeader(CONTENT_TYPE, "text/n3");
+        post.setEntity(new StringEntity(body));
+
+        assertEquals("Unable to create memento with RFC-1123 date-time format!",
+                CREATED.getStatusCode(), getStatus(post));
+
+        // Create memento with RFC-1123 date-time format in wrong value
+        final String dateTime1 = "Tue, 13 Jun 2008 11:05:30 ANYTIZONE";
```

Comment:
Could you slightly change the timestamp by a few seconds so its not identical to the first memento created?  In the unlikely event this got past the parsing step the request would be rejected for having a duplicate memento path.

Reply:
Also, up to you if you want to change this from `ANYTIZONE` to `ANYTIMEZONE` for clarity, just wanted to point it out in case it was unintentional.